amethyst_renderer/src/formats/texture.rs

```diff
@@ -7,71 +7,106 @@ use amethyst_assets::{
     ProgressCounter, Result, ResultExt, SimpleFormat,
 };
 use amethyst_core::specs::prelude::{Entity, Read, ReadExpect};
-use gfx::format::{ChannelType, SurfaceType};
+use gfx::format::{ChannelType, SurfaceType, SurfaceTyped};
 use gfx::texture::SamplerInfo;
 use gfx::traits::Pod;
-use tex::{Texture, TextureBuilder};
+use tex::{FilterMethod, Texture, TextureBuilder};
+use types::SurfaceFormat;
 use Renderer;
 
-/// Texture metadata, used while loading
+/// Additional texture metadata that can be passed to the asset loader or added to the prefab.
+///
+///
```

Comment:
2 empty lines?

Reply:
Don't know how they got there. I'm myself allergic to those ;P